src/sanitize.js

```diff
@@ -8,7 +8,10 @@ const clean = dirty => sanitizeHtml(dirty, {
 export const sanitizeElementName = clean;
 export const sanitizeAttrVal = clean;
 export const sanitizeAttr = clean;
-export const sanitizeJSON = dirty => JSON.stringify(dirty).replace(/</g, '\\u003c');
+export const sanitizeJSON = (dirty) => {
+  const result = JSON.stringify(dirty);
```

Comment:
@gilbox the `stringify` could be slow.  If we require this option to be an object we shouldn't have to worry about html injection.  I am not certain how end-user content could get in this value and cause injection.